pkg/common/catalog/catalog.go

```diff
@@ -209,6 +209,11 @@ func (c *catalog) startPlugins() error {
 
 func (c *catalog) configurePlugins() error {
 	for _, p := range c.plugins {
+		if !p.Config.Enabled {
+			c.l.Infof("%s plugin %s is disabled and will not be configured", p.Config.PluginType, p.Config.PluginName)
```

Comment:
Looks like for each disabled plugin, we'll receive several log lines?

I think `debug` level makes sense here. In the future, when we have statically-linked plugins, disabled will probably be the default state

Reply:
Right, I'll change it to `debug` level. Thanks!